### DISCUSSION TITLE
php5.6 curl的SSL Version和 openssl的OpenSSL Library Version 版本号不一致

### DISCUSSION THREAD

**Discussion by tombiran** [1Panel-dev/1Panel#7071] [Q&A]
### 联系方式

tombiran8@gmail.com

### 1Panel 版本

v1.10.20-lts

### 问题描述

php5.6 curl的SSL Version和 openssl的OpenSSL Library Version 版本号不一致 会导致curl请求https的时候 php异常退出 502   在phpinfo中可以查看版本
![photo_2024-11-15_06-58-42](https://github.com/user-attachments/assets/e6057a5e-fd76-47e9-83e8-0f61f82323e6)
![photo_2024-11-15_06-58-49](https://github.com/user-attachments/assets/808ef403-e4ab-4f81-83d2-40159d12a96a)

### 重现步骤

不会弄

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，已在工单中回复该问题，麻烦配合提供远程环境。

**Reply by hhrensheng** [reply]
最新的1.10.22 还有这个问题，麻烦解决一下啊

**Reply by wanghe-fit2cloud** [reply]
issue https://github.com/1Panel-dev/1Panel/issues/7652
